src/components/page/page.stories.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import contentSvg from '../../../src-docs/src/images/content.svg';
+import sideNavSvg from '../../../src-docs/src/images/side_nav.svg';
+
+import { EuiImage } from '../image';
+import { EuiPageSidebar } from './page_sidebar';
+import { EuiPageBody } from './page_body';
+import { EuiPageSection } from './page_section';
+import { EuiPage, EuiPageProps } from './page';
+
+const meta: Meta<EuiPageProps> = {
+  title: 'EuiPage',
+  component: EuiPage,
+};
+
+export default meta;
+type Story = StoryObj<EuiPageProps>;
+
+const componentDefaults: EuiPageProps = {
+  paddingSize: 'none',
+  grow: true,
+  direction: 'row',
+  restrictWidth: false,
+};
+
+export const Playground: Story = {
+  args: {
+    ...componentDefaults,
+  },
```

Comment:
I really like this declaration of `componentDefaults` to make it clear what we're setting!

That being said, if we're not setting any other props/args, there's no need to define a new obj/spread, we can just use `componentDefaults` as is:

```suggestion
  args: componentDefaults,
```

\+ feedback applies to all other components in the PR